### DISCUSSION TITLE
[Script request] auxxxilium (redpill) Xpenology

### DISCUSSION THREAD

**Discussion by gianlucaf81** [community-scripts/ProxmoxVE#262] [Request script]
### Application Name

Xpenology (synology emulation)

### Website

https://github.com/AuxXxilium

### Description

Arc is a customized Redpill Loader for DSM 7.x (Xpenology) with enhanced Hardwaresupport, Addons, guided (semi-automated) Installation and more. Multiple customization Options are built-in. It is modified to run on different Hardware (see Wiki for more Informations).

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Basically, this only makes sense as a VM. But I'm not familiar with synology etc. at all.

There used to be a PR, but it was never finished.
https://github.com/community-scripts/ProxmoxVE/pull/1622

I'll put the script in the pipeline on hold

**Comment by gianlucaf81** [comment]
thanks, you can see a step by step guide here if it helps. watch?v=vuj-Mu828-I

I made a little script to automatically download the compressed img file, it's a starting point but I don't know how to go beyond that.

The final script should:
- create the VM
- download the zip file containing arc.img
- import the arc.img file into the vm
- allow the user to choose the disks to use on the VM
- clean all temp file

[dwnl.sh.txt](https://github.com/user-attachments/files/19649246/dwnl.sh.txt)
